packages/terra-heading/CHANGELOG.md

```diff
@@ -3,6 +3,8 @@ Changelog
 
 Unreleased
 ----------
+### Removed
```

Comment:
This is a breaking change.

Reply:
Fixed now. https://github.com/cerner/terra-core/pull/2211/commits/e688373e7c8eddfa09f8e8de1fa4e9ef8750426e